modules/cluster_estimation/cluster_estimation_worker.py

```diff
@@ -0,0 +1,35 @@
+"""
+Gets detections in world space and outputs estimations of objects
+"""
+import queue
+
+from utilities import manage_worker
+from . import cluster_estimation
+
+
+def cluster_estimation_worker(# TODO: Initialization variables
+                              input_queue: queue.Queue,
+                              output_queue: queue.Queue,
+                              worker_manager: manage_worker.ManageWorker):
+    """
+    Worker process.
+
+    model_path and save_name are initial settings.
+    input_queue and output_queue are data queues.
+    worker_manager is how the main process communicates to this worker process.
+    """
+    estimator = cluster_estimation.ClusterEstimation()  # TODO: Initialization variables
+
+    while not worker_manager.is_exit_requested():
+        worker_manager.check_pause()
+
+        input_data = input_queue.get()
+        if input_data is None:
+            continue
+
+        # TODO: When to override
```

Comment:
Probably some sort of communication from the decision worker... don't worry about it for now.